docs/modules/ROOT/pages/how-to/use-tilt.adoc

```diff
@@ -0,0 +1,179 @@
+
+To follow this guide, please make sure that you have the following tools installed:
+
+`tilt`:: The https://docs.tilt.dev/[Tilt documentation] explains how to install it in your operating system of choice.
+
+`oc`:: You can download the OpenShift command directly from {product}, selecting the help menu (marked as a question mark) and selecting the "Command line tools" entry.
+
+`docker`:: You can download it from https://www.docker.com.
+
+== Procedure
+
+This section explains the steps required to use Tilt with {product}.
+
+=== 1. Login to {product}
+
+Follow these steps to login to {product} on your terminal and to create a project:
+
+. Login to the {product} console on your web browser.
+. Click on your user name on the top right and select "Copy login command"
+. Click "Display token" and copy the login command shown in "Log in with this token"
+. Paste the `oc login` command on the terminal:
++
+[source,shell]
+--
+$ oc login --token=sha256~_xxxxxx_xxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxx-X --server=https://api.${zone}.appuio.cloud:6443
+--
+
+. Create a new project called "fortune-go"
++
+[source,shell]
+--
+$ oc new-project fortune-go
+Now using project "fortune-go" on server "https://api.${zone}.appuio.cloud:6443".
+--
+
+=== 2. Login to the {product} Container Registry
+
+Tilt uses Docker to build the container image, and to publish to a container registry. In this example we're going to use the {product} integrated container registry.
+
+. Login to the {product} container registry (as explained in xref:how-to/use-integrated-registry.adoc[this page]):
++
+[source,shell]
+--
+$ oc whoami -t | docker login registry.${zone}.appuio.cloud -u $(oc whoami) --password-stdin
+--
+
+=== 3. Clone the sample "Fortune in Go" project
+
+The "Fortune in Go" project provides a very simple Go application to start with. You can clone it from GitLab:
+
+[source,shell]
+--
+$ git clone https://gitlab.com/vshn/applications/fortune-go.git
+$ cd fortune-go
+--
+
+=== 4. Add a Deployment File
+
+Add a file named `deployment.yaml` at the root of the project with the following information:
+
+[source,yaml]
+--
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fortune-go
+spec:
+  template:
+    spec:
+      containers: # <1>
+      - image: registry.[ZONE].appuio.cloud/fortune-go/fortune-go:latest
+        imagePullPolicy: Always
+        name: fortune-container
+        ports:
+        - containerPort: 8080
+    metadata:
+      labels:
+        app: fortune-go
+  selector:
+    matchLabels:
+      app: fortune-go
+  strategy:
+    type: Recreate
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fortune-go
+spec:
+  ports:
+    - port: 8080
+      targetPort: 8080
+  selector:
+    app: fortune-go
+  type: ClusterIP
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-production
+  name: fortune-go
+spec:
+  rules: # <1>
+  - host: fortune.apps.[ZONE].appuio.cloud
```

Comment:
Also point to the zones list as with the language how-tos?

Reply:
Done